phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -4107,6 +4109,9 @@ protected PhoenixConnection upgradeSystemCatalogIfRequired(PhoenixConnection met
             metaConnection = addColumnsIfNotExists(metaConnection, PhoenixDatabaseMetaData.SYSTEM_CATALOG,
                 MIN_SYSTEM_TABLE_TIMESTAMP_5_2_0,
```

Comment:
Here MIN_SYSTEM_TABLE_TIMESTAMP_5_2_0 should be increased by 1 and then each addColumns above will decrease it by 1. This can lead to BackwardCompatibilityIT tests to fail, which we are seeing,

Reply:
Done, thank you!

Reply:
The tests are still failing for the same reason, but this time on the `PHYSICAL_TABLE_NAME` column (the very first one). Could you please check to see what I might have done wrong?

Reply:
Value of MIN_SYSTEM_TABLE_TIMESTAMP needs to be increased by 1 as well here :- https://github.com/apache/phoenix/blob/6125f0244ce69b86235c19b39c6abaf3095f7eab/phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataProtocol.java#L102